packages/@aws-cdk/aws-appmesh/lib/virtual-router.ts

```diff
@@ -27,11 +27,6 @@ export interface IVirtualRouter extends cdk.IResource {
    * The  service mesh that the virtual router resides in
    */
   readonly mesh: IMesh;
-
-  /**
-   * Add a single route to the router
-   */
-  addRoute(id: string, props: RouteBaseProps): Route;
```

Comment:
Why was this removed?

Reply:
Will add back.  Removed originally because I thought, incorrectly, that you should not have this method on the Imported Virtual Router. This is valid though, as you can add a route to a virtual router that is not in your stack